README.md

```diff
@@ -1 +1,265 @@
+
+```sh
+$ builder run foo
+$ builder run bar
+```
+
+Sequentially via `||` or `&&` shell helpers:
+
+```sh
+$ builder run foo && builder run bar
+```
+
+Concurrently via the Builder built-in `concurrent` command:
+
+```sh
+$ builder concurrent foo bar
+```
+
+With `concurrent`, all tasks continue running until they all complete _or_
+any task exits with a non-zero exit code, in which case all still alive tasks
+are killed and the Builder process exits with the error code.
+
+## Archetypes
+
+Archetypes deal with common scenarios for your projects. Like:
+
+* [builder-react-component][]: A React component
+* A React application server
+* A Chai / jQuery / VanillaJS widget
+
+Archetypes typically provide:
+
+* A `package.json` with `builder`-friendly `script` tasks.
+* Dependencies and dev dependencies to build, test, etc.
+* Configuration files for all `script` tasks.
+
+In most cases, you won't need to override anything. But, if you do, pick the
+most granular `scripts` command in the archetype you need to override and
+define _just that_ in your project's `package.json` `script` section. Copy
+any configuration files that you need to tweak and re-define the command.
+
+### Task Resolution
+
+The easiest bet is to just have _one_ archetype per project. But, multiple are
+supported. In terms of `scripts` tasks, we end up with the following example:
+
+```
+ROOT/package.json
+ROOT/node_modules/ARCHETYPE_ONE/package.json
+ROOT/node_modules/ARCHETYPE_TWO/package.json
+```
+
+Say we have a `.builderrc` like:
+
+```yaml
+---
+archetypes:
+  - ARCHETYPE_ONE
+  - ARCHETYPE_TWO
+```
+
+The resolution order for a `script` task (say, `foo`) present in all three
+`package.json`'s would be the following:
+
+* Look through `ROOT/package.json` then the configured archetypes in _reverse_
+  order: `ARCHETYPE_TWO/package.json`, then `ARCHETYPE_ONE/package.json` for
+  a matching task `foo`
+* If found `foo`, check if it is a "passthrough" task, which means it delegates
+  to a later instance -- basically `"foo": "builder run foo"`. If so, then look
+  to next instance of task found in order above.
+
+### Special Archetype Tasks
+
+Archetypes use conventional `scripts` task names, except for the following
+special cases:
+
+* `"npm:postinstall"`
+* `"npm:preversion"`
+* `"npm:version"`
+* `"npm:test"`
+
+These tasks are specifically actionable during the `npm` lifecycle, and
+consequently, the archetype mostly ignores those for installation by default,
+offering them up for actual use in _your_ project.
+
+As an **additional restriction**, non-`npm:FOO`-prefixed tasks with the same
+name (e.g., `FOO`) _may_ call then `npm:`-prefixed task, but _not_ the other
+way around. So
+
+```js
+// Good / OK
+"npm:test": "builder run test-frontend",
+"test": "builder run npm:test",
+
+// Bad
+"npm:test": "builder run test",
+"test": "builder run test-frontend",
+```
+
+## Package Script Commands
+
+The underyling concept here is that `builder` `script` commands simple _are_
```

Comment:
Thanks -- I'm moving around a lot of the documentation, so this comment will disappear, but I fixed the spelling.